src/components/editor/Editor.tsx

```diff
@@ -95,15 +95,15 @@ export const Editor = ({
     !templateMetadata ||
     !document ||
     !visualConfigurationDataFetched ||
-    !themeDataFetched;
+    (templateMetadata?.themeEntityId && !themeDataFetched);
```

Comment:
I still feel like we should set themeDataFetched to true even if there's no theme data to fetch so we don't have to check multiple things in multiple places. This makes it more confusing IMO because it requires us to understand more about the intricacies of how the theme stuff works. As a counter example, we don't check multiple things for vcData, we just check that it was fetched.